lib/aida/message.ex

```diff
@@ -258,4 +258,8 @@ defmodule Aida.Message do
   def mark_sensitive(message) do
     %{message | sensitive: true}
   end
+
+  def is_unsubscribe_keyword(message) do
+    text_content(message) == message.bot.front_desk.unsubscribe_keyword[language(message)]
```

Comment:
Such decomposition on the bot/front_desk structure indicates that this should be bot's (or even front_desk's) responsibility. Maybe asking for the content of the message from the bot/front_desk side and move the comparison there?